### ISSUE TITLE
4- and 6-Digit HUCs have the same geometry in QRV

### ISSUE THREAD

**Issue by shelbysawyer** [Riverscapes/RiverscapesXML#569]

**Comment by joewheaton** [comment]
Good catch and known issue. @jtgilbert explained that this is the case for any six digit HUC that ends in 00. Odd, but it is correct.

**Comment by jtgilbert** [comment]
Yeah, I wouldn't even call it an issue, just a simple fact that in some cases the HUC4 and 6 are identical.
